app/models/chat_channel.rb

```diff
@@ -35,6 +35,10 @@ def group?
     channel_type != "direct"
   end
 
+  def private_org_channel?
+    channel_name.to_s.ends_with?(" private group chat") #e.g. @devteam private group chat
```

Comment:
Basically determining functionality from matching a name. Possibly cleaner to do it otherwise, but this keeps our optionality open for how we might want to handle different contexts. I may push a different approach soon.